cloud_functions/token_refresh/streams/strava.py

```
@@ -0,0 +1,92 @@
+    Raises:
+        ValueError: If refresh token is invalid.
+    """
+    encryption_key = get_secret(os.environ["ENCRYPTION_SECRET"])["token"]
+
+    client = Client()
+    strava_secret = get_secret(environ["STRAVA_SECRET"])
+    client_id = strava_secret.get("client_id")
+    client_secret = strava_secret.get("client_secret")
+
+    if not client_id or not client_secret:
+        raise ValueError("Strava credentials not found in Secret Manager")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)